src/Hedgehog.Experimental.Tests/GenTests.fs

```diff
@@ -249,24 +251,26 @@ let ``increasing4 generates a 4-tuple with strictly increasing elements`` () =
         x3 <! x4
     }
 
+let dateTimeRange = Range.constantFrom (DateTime (2000, 1, 1)) DateTime.MinValue DateTime.MaxValue
```

Comment:
Thanks! But I'd prefer to have an exponential range here. Something like:

```f#
Range.exponentialFrom
  (DateTime(2000, 1, 1)).Ticks
  DateTime.MinValue.Ticks
  DateTime.MaxValue.Ticks
|> Range.map DateTime)
```

(Format it as you see fit.)

Reply:
I agree; Exponential with DateTime makes way more sense than Constant.